superset-frontend/src/common/components/Collapse.tsx

```diff
@@ -68,6 +90,18 @@ const Collapse = Object.assign(
         }
       }
     }
+    .ant-collapse-item-active {
+      .ant-collapse-header {
+        ${({ expandIconPosition }) =>
+          expandIconPosition &&
+          expandIconPosition === 'right' &&
+          `
+            .anticon.anticon-right.ant-collapse-arrow > svg {
+              transform: rotate(-90deg) !important;
```

Comment:
can we avoid using important?

Reply:
Nope. This is added by the library on the element itself, which makes the important necessary in order to override